### DISCUSSION TITLE
Does this SDK support Server 5.6, 5.7 and 8.x?

### DISCUSSION THREAD

**Discussion by tsaiggo** [mysql-net/MySqlConnector#1307] [Q&A] [answered]
Hi guys, I have a little question. Does this SDK support MySQL server 5.6, 5.7 and 8.x?
I did not find any evidence in the documentation regarding which MySQL server versions are supported by the SDK.

**Comment by bgrainger** [comment, marked as answer]
Yes: every build runs integration tests against MySQL 5.6, 5.7, and 8.0.

Created https://github.com/mysql-net/MySqlConnector/issues/1308 to make this information more prominent.
